apstools/utils/catalog.py

```diff
@@ -26,46 +34,34 @@
 logger = logging.getLogger(__name__)
 
 
-def copy_filtered_catalog(source_cat, target_cat, query=None):
+def copy_filtered_catalog(
+    source_cat: Union[Broker, Any],
+    target_cat: Union[Broker, Any],
+    query: Optional[dict[str, Any]] = None,
+) -> None:
     """
-    copy filtered runs from source_cat to target_cat
+    Copy filtered runs from source_cat to target_cat.
 
-    PARAMETERS
-
-    source_cat
-        *obj* :
-        instance of `databroker.Broker` or `databroker.catalog[name]`
-    target_cat
-        *obj* :
-        instance of `databroker.Broker` or `databroker.catalog[name]`
-    query
-        *dict* :
-        mongo query dictionary, used to filter the results
-        (default: ``{}``)
-
-        see: https://docs.mongodb.com/manual/reference/operator/query/
-
-    example::
-
-        copy_filtered_catalog(
-            databroker.Broker.named("mongodb_config"),
-            databroker.catalog["test1"],
-            {'plan_name': 'snapshot'})
+    Args:
+        source_cat: instance of `databroker.Broker` or `databroker.catalog[name]`
+        target_cat: instance of `databroker.Broker` or `databroker.catalog[name]`
+        query: mongo query dictionary to filter results (default: None)
     """
     query = query or {}
     for i, uid in enumerate(source_cat.v2.search(query)):
         run = source_cat.v1[uid]
-        # fmt: off
-        logger.debug(
-            "%d  %s  #docs=%d", i + 1, uid, len(list(run.documents()))
-        )
-        # fmt: on
+        logger.debug("%d  %s  #docs=%d", i + 1, uid, len(list(run.documents())))
```

Comment:
This line is long and hard to read.  Use a trailing `,` and ruff will make it multiline.
```suggestion
        logger.debug("%d  %s  #docs=%d", i + 1, uid, len(list(run.documents())),)
```